util/dvsim/FlowCfg.py

```diff
@@ -101,6 +101,8 @@ def __init__(self, flow_cfg_file, hjson_data, args, mk_config):
 
         # Full results in md text
         self.results_md = ""
+        # Results in hjson
+        self.results_hjson = {}
```

Comment:
Can I suggest that these default to `None`? That will make it a bit more robust if something goes wrong and we end up serializing as `{}` (which I think might happen if the testpoints list is empty)